City-of-Helsinki/drupal-asuntotuotanto
# React package 1.3.6 and composer update

### Description

React package update 1.3.4 to 1.3.6 and composer helfi_ package updates


### How to test it

- make fresh
- login as admin
- go re-index apartment search api indexes https://asuntotuotanto.docker.so/fi/admin/config/search/search-api
- check that haso ja hitas listing pages are working
  - https://asuntotuotanto.docker.so/fi/asuntohaku/hitas
  - https://asuntotuotanto.docker.so/fi/asuntohaku/haso
